Taskfile.yml

```diff
@@ -41,15 +41,18 @@ vars:
   # build vars
   COMMIT:
     sh: echo ${TRAVIS_COMMIT:-`git log -n 1 --format=%h`}
+  TAG:
+    sh: echo `git describe --tags --abbrev=0`
```

Comment:
This command gets the most recent tag (don't know if it's an expected behaviour)
https://stackoverflow.com/questions/1404796/how-to-get-the-latest-tag-name-in-current-branch-in-git

It depends on the git development strategy used